From a91d2bd9240202fc3c35c6bdd1dae0571a61d438 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Mon, 10 Oct 2016 15:03:38 +0800
Subject: [PATCH] regulator: mp8865: add set_voltage_time_sel func

support delay time in microseconds required to
rise or fall to this new voltage

Change-Id: I8d096500a3dcb376785285d08228961cf6b26ce0
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/regulator/mp8865-regulator.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/regulator/mp8865-regulator.c b/drivers/regulator/mp8865-regulator.c
index ec8b63baef73..416333221d13 100644
--- a/drivers/regulator/mp8865-regulator.c
+++ b/drivers/regulator/mp8865-regulator.c
@@ -93,6 +93,7 @@ static struct regulator_ops mp8865_dcdc_ops = {
 	.map_voltage = regulator_map_voltage_linear_range,
 	.get_voltage_sel = regulator_get_voltage_sel_regmap,
 	.set_voltage_sel = mp8865_set_voltage,
+	.set_voltage_time_sel = regulator_set_voltage_time_sel,
 	.enable = regulator_enable_regmap,
 	.disable = regulator_disable_regmap,
 	.is_enabled = regulator_is_enabled_regmap,
-- 
2.35.3

